infrastructure/dogfood/terraform/aws-tf-module/main.tf

```diff
@@ -259,3 +259,8 @@ module "notify_slack" {
   slack_channel     = "#help-p1"
   slack_username    = "monitoring"
 }
+
+module "ses" {
+  source = "github.com/fleetdm/fleet//terraform/addons/ses?ref=edwardsb-ses-mailer"
```

Comment:
Is the plan to add a commit to change this to `main` when that branch is ready to go to main?

Reply:
yes